v4l2/v4l2_codecbase.cpp

```diff
@@ -279,12 +281,13 @@ const char* V4l2CodecBase::IoctlCommandString(int command)
 }
 #endif
 
-int32_t V4l2CodecBase::ioctl(int command, void* arg)
+int32_t V4l2CodecBase::ioctl(uint64_t command, void* arg)
```

Comment:
step 2 is not must, but nice to have.
you can update intel code only in vaapi_V4L2_video_device.h/cc, since it's safe to cast chrome's   'int' to intel's 'int64_t'.